src/css/styles.scss

```diff
@@ -171,14 +193,25 @@ main {
   margin-bottom: .2vw;
 }
 
+.hydration-weekly-avg,
+.hydration-week-data-list {
+  text-align: center;
+  width: 100%;
+}
+
 .trend-line {
   font-size: 2vw;
   text-align: center;
   width: 20vw;
 }
 
 /* BUTTONS */
-.info-button {
+.info-button,
+.friends-button,
+.calendar-button,
```

Comment:
So with the SCSS lesson in mind, are we going to need to go back and refactor our file?